### ISSUE TITLE
Can you provide a script on training the model and image retrieval?

### ISSUE THREAD

**Issue by BenjiKCF** [vidarlab/xformer-paired-viz#2]
I would like to publish a paper based on this topic. Can you provide a script for training the model? Thank you so much.

**Comment by jena-shreyas** [comment]
Yes please, @sblack15, I am trying to reproduce your results for other image retrieval datasets, and it would be really helpful if you could provide the training parameters for the ViT-B16 and ResNet-101 models.
